test/transaction.cpp

```diff
@@ -0,0 +1,163 @@
+        BOOST_REQUIRE(tObj.count("value") > 0);
+        BOOST_REQUIRE(tObj.count("gasPrice") > 0);
+        BOOST_REQUIRE(tObj.count("gasLimit") > 0);
+        BOOST_REQUIRE(tObj.count("nonce")> 0);
+        BOOST_REQUIRE(tObj.count("to") > 0);
+
+        BOOST_REQUIRE(tObj.count("v") > 0);
+        BOOST_REQUIRE(tObj.count("r") > 0);
+        BOOST_REQUIRE(tObj.count("s") > 0);
+
+        //Construct Rlp of the given transaction
+        RLPStream rlpStream;
+        rlpStream.appendList(9);
+        rlpStream << toInt(tObj["nonce"]) << toInt(tObj["gasPrice"]) << toInt(tObj["gasLimit"]);
+        if (tObj["to"].get_str().empty())
+            rlpStream << "";
+        else
+            rlpStream << Address(tObj["to"].get_str());
+        rlpStream << toInt(tObj["value"]) << importData(tObj);
+
+        u256 r = h256(fromHex(tObj["r"].get_str()));
+        u256 s = h256(fromHex(tObj["s"].get_str()));
+        rlpStream << (toInt(tObj["v"]) + 27) << r << s;
+
+        Transaction tr(rlpStream.out(), true);
+        if(_fillin)
+        {
+               o["rlp"] = "0x" + toHex(tr.rlp());
+               o["sender"] = toString(tr.sender());
+        }
+
+        BOOST_REQUIRE(o.count("rlp") > 0);
+        BOOST_REQUIRE(o.count("sender") > 0);
+        bytes rlpReaded = importByteArray(o["rlp"].get_str());
+        Address addressReaded = Address(o["sender"].get_str());
+
+        //cnote << "Readed RLP: " << toHex(rlpReaded);
+        //cnote << "Sender: " << tr.sender();
+        //cnote << "Transaction RLP: " << toHex(tr.rlp());
+
+        Transaction trFromRLP = Transaction(rlpReaded);
+
+        //Check the fields restored from RLP to original fields
+        BOOST_CHECK_MESSAGE(tr.data() == trFromRLP.data(), "Data in given RLP not matching the Transaction data!");
+        BOOST_CHECK_MESSAGE(tr.value() == trFromRLP.value(), "Value in given RLP not matching the Transaction value!");
+        BOOST_CHECK_MESSAGE(tr.gasPrice() == trFromRLP.gasPrice(), "GasPrice in given RLP not matching the Transaction gasPrice!");
+        BOOST_CHECK_MESSAGE(tr.gas() == trFromRLP.gas(),"Gas in given RLP not matching the Transaction gas!");
+        BOOST_CHECK_MESSAGE(tr.nonce() == trFromRLP.nonce(),"Nonce in given RLP not matching the Transaction nonce!");
+        BOOST_CHECK_MESSAGE(tr.receiveAddress() == trFromRLP.receiveAddress(), "Receive address in given RLP not matching the Transaction 'to' address!");
+        BOOST_CHECK_MESSAGE(tr.sender() == trFromRLP.sender(), "Transaction sender address in given RLP not matching the Transaction 'vrs' signature!");
+        BOOST_CHECK_MESSAGE(tr.sender() ==  addressReaded || trFromRLP.sender() == addressReaded, "Signature address of sender does not match given sender address!");
+    }
+}
+} }// Namespace Close
+
+
+BOOST_AUTO_TEST_SUITE(TransactionTests)
+
+/*BOOST_AUTO_TEST_CASE(ttExample)
+{
+    dev::test::executeTests("ttExample", "/TransactionTests", dev::test::doTransactionTests);
+}*/
```

Comment:
you can comment that in. This is the right way to do it. ttCreateTest is only for experimenting with single tests. I want to be able to update all tests with ./testeth --filltests and test all tests with ./testeth.